Project-Descriptions-and-Plans/CVE20/CVE20-PA-2024.md

```diff
@@ -121,18 +121,15 @@ Companies developing microcontroller-based embedded (sub)systems or devices.
 *This section provides the detailed feature list of the project.*
```

Comment:
Many of these comments in italics seem to be instructions for the person creating/maintaining this document.  As such, they probably should not be rendered to html for the general reader see.  Recommend we make these Markdown comments so that the editor of this document can see them, but the general reader would not.  For example:
```
<!---
Note to editor:
If the features are outlined in detail in the PL, or within a User Manual or other document, refer to that document. If not, the feature list should be provided here.
--->
```

Reply:
ok, I didn't know about this technique.

Reply:
changed to identify this as a risk/deficiency in the plan that OpenHW staff are working to alleviate.